mix.exs

```diff
@@ -7,15 +7,21 @@ defmodule Krakex.Mixfile do
     [
       app: :krakex,
       version: @version,
-      elixir: "~> 1.4",
+      elixir: "~> 1.8",
```

Comment:
This is to ensure the minimum version of elixir that is supported so I'd like to keep this at 1.4 as long as that's officially supported by the community.